### PR TITLE
New time conductor popups should be closed by hitting ESC key

### PR DESCRIPTION
**Pull request by kobe1104**
Addressing issue https://github.com/nasa/openmct/issues/3976

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes? N/A
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Testing instructions:
1. At openmct select realtime mode by switching to 'Local Clock' at time conductor.
2. Click on either start time or end time to open the time input popup.
3. Press the ESC key button and observe that the time input popup is closed now.

### CODE REVIEW

**Comment by charlesh88** [comment]
@kobe1104 Awesome!

**Review by nikhilmandlik — APPROVED** [review]
LGTM

File: src/plugins/timeConductor/timePopup.vue [hunk 1]
```diff
@@ -3,6 +3,8 @@
     class="pr-tc-input-menu"
     @keydown.enter.prevent
     @keyup.enter.prevent="submit"
+    @keydown.esc.prevent
+    @keyup.esc.prevent="hide"
```
Comment (nikhilmandlik) [inline src/plugins/timeConductor/timePopup.vue:7]: 👍

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue? | Y
Appropriate unit tests included? | NA
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) | Y

**Pull request merged**